integration_test/install_test.go

```diff
@@ -216,3 +220,14 @@ func localTestServer() (string, func()) {
 	s := httptest.NewServer(http.FileServer(http.Dir("testdata")))
 	return s.URL, s.Close
 }
+
+func assertManifestPluginIndex(t *testing.T, test *ITest, plugin string) {
```

Comment:
assertPluginFromIndex is a better name potentially?
also why is this not defined on `test` (like assertExecutableInPath)? that way you don't have to pass it `t`.

Reply:
I can add it there. I added this to just check "detached" and thought that would be a little too specific to `install --manifest` to add to `testutil_test.go`.